src/benchmarks.zig

```diff
@@ -1,349 +1,538 @@
 const std = @import("std");
 const builtin = @import("builtin");
-
 const sig = @import("sig.zig");
+const pt = @import("prettytable");
+const math = std.math;
 
 const Decl = std.builtin.Type.Declaration;
+const Duration = sig.time.Duration;
 
-const io = std.io;
-const math = std.math;
-const meta = std.meta;
+pub const BenchTimeUnit = enum {
+    Nanos,
+    Millis,
+    Seconds,
+
+    pub fn convertDuration(self: BenchTimeUnit, duration: Duration) u64 {
+        return switch (self) {
+            .Nanos => duration.asNanos(),
+            .Millis => duration.asMillis(),
+            .Seconds => duration.asSecs(),
```

Comment:
Why the titlecase enum members?

Reply:
706be88d0c5d07fcc9804caa954076625dba552f